tests/README.md

```diff
@@ -1,99 +1,39 @@
 
 ## Prerequisites
-Install mage using the instructions
-[here](https://github.com/magefile/mage#installation)
+Install kubectl using the instructions [here](https://kubernetes.io/docs/tasks/tools/install-kubectl) 
 
-Install kubectl using the instructions
-[here](https://kubernetes.io/docs/tasks/tools/install-kubectl) 
+The tests use Kustomize to deploy the cass-operator. Kustomize is installed if not present in the machine. We test against 4.1.x series of Kustomize at this point, but newer 3.x releases should work also. 
 
-## Running the tests
-The tests themselves expect a running k8s cluster, with at least 6 worker nodes,
-and kubectl to be configured to point at the cluster.
-
-You can either stand up your own cluster and point kubectl to it yourself, or use
-a mage target to bootstrap and configure a cluster for you.
-
-Refer to the [k8s targets](../docs/developer/k8s_targets.md) for more information on
-supported k8s flavors.
-
-### Using preconfigured targets for cluster management
-
-To let a mage bootstrap a new cluster, run all of the tests,
-and then tear down the cluster at the end, run the `k8s:runIntegTests` target
-with the desired k8s flavor specified:
-
-#### Using a k3d cluster
-```
-M_K8S_FLAVOR=k3d mage k8s:runIntegTests
-```
-
-#### Using a KIND cluster
-```
-M_K8S_FLAVOR=kind mage k8s:runIntegTests
-```
-
-If you just want to have the cluster stood up and configured for you, but
-not automatically run the tests or tear down, run:
+You will also need a running Kubernetes cluster. We use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation), but others should work also. Note that the storageClass is set to ``default`` in the tests.
 
-#### Using a k3d cluster
-```
-M_K8S_FLAVOR=k3d mage k8s:setupEmptyCluster
-```
-
-### Using a custom Docker registry for the operator image
-
-If M_DOCKER_USERNAME, M_DOCKER_PASSWORD, and M_DOCKER_SERVER environment variables are defined, then they will be used to create an image pull secret.  This secret will automatically be used by the integration tests for pulling the Docker image of the operator.
+## Running the tests
+The tests themselves expect a running k8s cluster, with at least 6 worker nodes, and kubectl to be configured to point at the cluster.
 
-This should be used in conjunction with the M_OPERATOR_IMAGE environment variable to select a specific image.  Note that the M_OPERATOR_IMAGE value should include the name of the custom registry.
+### Running tests with custom operator image
 
-Example:
+If you wish to use a custom operator image (or deploy from custom container registry), modify the used image by running:
 
-```console
-export M_DOCKER_USERNAME=USERNAME
-export M_DOCKER_PASSWORD=ACCESSTOKEN
-export M_DOCKER_SERVER="docker.pkg.github.com"
-export M_OPERATOR_IMAGE="docker.pkg.github.com/datastax/cass-operator/operator:latest-ubi"
+```bash
+cd config/manager && kustomize edit set image controller=$(IMG)
```

Comment:
I think `$(IMG)` should be `$IMG`.

Is this really how we to configure a custom image? This is going to change `config/manager/kustomization.yaml` which is stored in git.

Reply:
Yes, that's how kustomize works. I'm not entirely sure in which cases one would use a custom image with tests that's not built from the repository and then commit something. 

Otherwise make deploy takes as input the IMG.